src/appPackageManifest.ts

```
@@ -77,6 +84,25 @@ class AppPackageManifestTransform extends Transform {
     super({ objectMode: true });
   }
 
+  private populateTileData(): void {
+    if (this.projectConfig.appType === AppType.APP) {
```

It looks like tile info will be silently ignored if the appType isn't `APP`. Add an error in the config validator to catch this case?

---

Sure. Will add a warning in the validator.